src/SearchParameters/index.js

```diff
@@ -570,7 +570,7 @@ SearchParameters.prototype = {
    */
   getExcludeRefinements: function(facetName) {
     if (!this.isConjunctiveFacet(facetName)) {
-      throw new Error(facetName + ' is not defined in the facets attribute of the helper configuration');
+      return [];
```

Comment:
Not sure the branch is still useful. The return value already fallback on `[]`.

Reply:
not sure, but this is the most consistent way, all getXXXrefinemnent will use the fallback if it's not declared